workflows/variantcall.nf

```diff
@@ -41,37 +39,23 @@ workflow VARIANTCALL {
     //     params.chain_checksum
     // )
 
-    // SanityCheck
-    sanityCheck(
+    // Prepare metadata, adapted from SanityCheck
+    PREP_META(
         file(params.experiment_info_tsv),
         params.api_token,
         params.song_url,
         params.clinical_url,
         params.skip_duplicate_check
```

Comment:
Add conf parameters to symlink output e.g. 
https://github.com/icgc-argo-workflows/dnaaln/blob/version1.0.0/conf/icgc-argo-workflows/modules.config#L25-L33